firmware/ventilator-controller-stm32/CMakeLists.txt

```diff
@@ -38,15 +38,44 @@ include_directories(
 )
 
 if ("${CMAKE_BUILD_TYPE}" STREQUAL "TestCatch2")
+    set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/)
+    include(CodeCoverage)
+    append_coverage_compiler_flags()
+
+    # disable all optimization
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O0")
+    set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -O0")
+    set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fno-inline -fno-inline-small-functions -fno-default-inline")
+
+    # Some ARM header files give conversion errors from 32-bit to 64-bit
+    # TODO: Need t figure out how to enable this for a small section of code
+    # instead of for the whole project
+    set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fpermissive")
```

Comment:
Is this flag still needed? / are we still pulling in ARM headers now that we've removed the STM32 headers?

Reply:
No! works fine without it now. removing